test/format/salvage.c

```diff
@@ -164,14 +164,7 @@ wts_salvage(void)
 	wts_verify("post-salvage verify");
 	wts_close();
 
-	/*
-	 * If no records were deleted, dump and compare against Berkeley DB.
-	 * (The problem with deleting records is salvage restores deleted
-	 * records if a page splits leaving a deleted record on one side of
-	 * the split, so we cannot depend on correctness in that case.)
-	 */
-	if (g.c_delete_pct == 0)
-		wts_dump("salvage", SINGLETHREADED);
+	wts_dump("salvage", USE_BERKELEY_DB);
```

Comment:
I think we should remove the second parameter to `wts_dump` and instead have `if (!USE_BERKELEY_DB) return;` near the start of that function.

There is no point calling out to the `s_dumpcmp.sh` shell script if Berkeley DB is not enabled.